source/apiVolontaria/apiVolontaria/tests/tests_view_Users.py

```diff
@@ -504,6 +503,10 @@ def test_list_users(self):
 
         self.assertEqual(json.loads(response.content)['count'], 2)
 
+        response = self.client.get(reverse('users')+"?search=%s" % self.user.email)
+
+        self.assertEqual(json.loads(response.content)['count'], 1)
+
```

Comment:
This test is good but please just create a new one so we have a test for basic list and for filtered list

Reply:
Of course